.changeset/cuddly-hats-swim.md

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
This PR doesn't need a changeset unless you believe that this should be released asap~. From my PoV this change can just be included in the next release without being mentioned explicitly.

Reply:
I still think it's good to do so that the changes propagate up to `npmjs.com`.